test/e2e/framework/pod/pod_client.go

```diff
@@ -192,6 +207,19 @@ func (c *PodClient) DeleteSync(ctx context.Context, name string, options metav1.
 	framework.ExpectNoError(WaitForPodNotFoundInNamespace(ctx, c.f.ClientSet, name, c.namespace, timeout), "wait for pod %q to disappear", name)
 }
 
+// addTestOrigin adds annotations to help identifying tests which incorrectly leak pods because insufficient cleanup
+func (c *PodClient) setOwnerAnnotation(pod *v1.Pod) {
+	ginkgo.GinkgoHelper()
+	if !c.trackOwner {
+		return
+	}
```

Comment:
Nit: you can return before the `GinkgoHelper` call.

Reply:
thanks, will do